### PR TITLE
RealmResults.isLoaded description fix

### LINKED ISSUE #2895: RealmResults.isLoaded() method description is incorrect.
RealmResults.isLoaded() method description is incorrect.
`Returns true if the results are not yet loaded` ???

### PR DESCRIPTION
**Pull request by stk1m1**
`RealmResults.isLoaded()` description fix. Closes #2895.

@realm/java

### CODE REVIEW

File: realm/realm-library/src/main/java/io/realm/RealmResults.java [hunk 1]
```diff
@@ -851,12 +851,12 @@ void setPendingQuery(Future<Long> pendingQuery) {
     }
 
     /**
-     * Returns {@code true} if the results are not yet loaded, {@code false} if they are still loading. Synchronous
+     * Returns {@code false} if the results are not yet loaded, or they are still being loaded. Synchronous
```
Comment (cmelchior) [inline realm/realm-library/src/main/java/io/realm/RealmResults.java]: Returns {@code false} if the results are not yet loaded, {@code true} if they are loaded.

File: realm/realm-library/src/main/java/io/realm/RealmResults.java [hunk 2]
```diff
@@ -851,12 +851,12 @@ void setPendingQuery(Future<Long> pendingQuery) {
     }
 
     /**
-     * Returns {@code true} if the results are not yet loaded, {@code false} if they are still loading. Synchronous
+     * Returns {@code false} if the results are not yet loaded or still being loaded. Synchronous
```
Comment (cmelchior) [inline realm/realm-library/src/main/java/io/realm/RealmResults.java]: Returns {@code true} if the results are loaded and available, {@code false} if they are still loading.

File: realm/realm-library/src/main/java/io/realm/RealmResults.java [hunk 3]
```diff
@@ -851,12 +851,12 @@ void setPendingQuery(Future<Long> pendingQuery) {
     }
 
     /**
-     * Returns {@code true} if the results are not yet loaded, {@code false} if they are still loading. Synchronous
+     * Returns {@code false} if the results are not yet loaded, {@code true} if they are loaded. Synchronous
      * query methods like findAll() will always return {@code true}, while asynchronous query methods like
      * findAllAsync() will return {@code false} until the results are available.
      * This will return {@code true} if called for an unmanaged object (created outside of Realm).
```
Comment (zaki50) [inline realm/realm-library/src/main/java/io/realm/RealmResults.java]: Can we remove this line?
Unmanaged RealmResults does not exist.

Comment (stk1m1) [inline realm/realm-library/src/main/java/io/realm/RealmResults.java]: Great point. I'll fix it.

**Comment by stk1m1** [comment]
@realm/ava reviews are addressed.

**Comment by zaki50** [comment]
:+1:, but you need to resolve conflicts before merging.

**Comment by zaki50** [comment]
I merged master

**Comment by cmelchior** [comment]
👍

**Comment by kneth** [comment]
:+1:

**Pull request merged**